controlplane/src/core/bufservices/subgraph/getSubgraphById.ts

```diff
@@ -49,6 +49,8 @@ export function getSubgraphById(
         namespace: subgraph.namespace,
         websocketSubprotocol: subgraph.websocketSubprotocol || '',
         isFeatureSubgraph: subgraph.isFeatureSubgraph,
+        baseSubgraphId: 'baseSubgraphId' in subgraph ? subgraph.baseSubgraphId : undefined,
+        baseSubgraphName: 'baseSubgraphName' in subgraph ? subgraph.baseSubgraphName : undefined,
```

Comment:
Not 100%, but I believe this is the equivalent of just `subgraph.baseSubgraphId`, if the field is not in `subgraph`, it is already undefined, no?

Reply:
We are expecting two different types for the subgraph. SubgraphDTO and FeatureSubgraphDTO. Both types contain the same fields except for the second type, which is a superset that also holds baseSubgraphId and baseSubgraphName. 

If you try to access those fields without casting or `in` you'll get an error